torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2993,8 +3040,9 @@ def create_symbol(
                 self.var_to_range[sympy_expr] &= constraint_dim.vr
```

Comment:
OK, I see you didn't modify the var_to_range code, it doesn't really make sense to have ranges for singletons, not sure if you stub things out there, is it possible to remove the >= tests? It would be nice if this was the case 🤔

Reply:
I'm not sure if this actually addreses your question, but I recently update singleton ints to error (instead of returning False) when doing inequalities: >=, etc. (it's the first PR in this stack) for an (un?)related reason. Some small exceptions are made however , e.g.  j0 >= c returns True when c is <= 2.

Reply:
I guess my question is, are you preventing var range tests on singleton ints? If so, we should avoid setting `var_to_range` for these.